src/main/kotlin/com/epam/brn/service/UserDetailsService.kt

```diff
@@ -12,15 +12,15 @@ class UserDetailsService(@Autowired val userDetailsDAO: UserDetailsRepository) {
     private val log = logger()
 
     fun getLevel(userId: String): Int {
-        log.info("current level = 1")
+        log.info("current progress = 1")
         return 1
     }
 
     fun addUser(name: String, email: String, phone: String): Int {
         // val newUser = UserDetails()
-        log.info("add new user $name $email $phone")
-        val newUser = userDetailsDAO.save(UserDetails(1, name, email, phone))
-        return newUser.id
+//        log.info("add new user $name $email $phone")
```

Comment:
I would remove all commented code and just leave a //TODO.

Reply:
totally agree